### ISSUE TITLE
Have problem to load_scene into sionna

### ISSUE THREAD

**Issue by as057923553** [NVlabs/sionna#625]
These following are error messages.
![image](https://github.com/user-attachments/assets/6a5c4823-96dc-4a85-8ab9-612d043cc578)
![image](https://github.com/user-attachments/assets/65b5846d-be23-4e53-a2c1-09da6ff72d5c)

These following are my files
![image](https://github.com/user-attachments/assets/9cdcf14d-c8cf-4949-9821-0f1b04472bc5)
![image](https://github.com/user-attachments/assets/41d8ebdd-b597-4cca-a7e3-3ebf11e7eedc)

These following are my Blender screen
![image](https://github.com/user-attachments/assets/70313ba8-d493-4c67-931c-b99444b3f1db)

Could anyone hlep me, please.

**Comment by Fedomer** [comment]
Hello,
1. install the last sionna (0.19)
2. Control the slashes in the path (you are in windows it seems!!!!) 
3. create your scene with the video tutorial on YouTUBE. [Create Blender Scene /OSM](https://www.youtube.com/watch?v=7xHLDxUaQ7c) and test if is all OK.
If it's you have created the scene in wrong way.

**Comment by merlinND** [comment]
Hello @as057923553,

On top of what @Fedomer suggested, could you try making sure that all your meshes and materials in Blender have names that only use alphanumeric characters? And then re-export and trying again.
In theory any characters should work, but it's worth trying, especially on Windows.

Please also make sure that the `meshes/` directory contains all meshes referenced in the XML file.

**Comment by as057923553** [comment]
Thanks for your help! I had already followed the youtube video to make it! However, I still have some problems.

@Fedomer The reason why I used sionna 0.14 was because this problem.
![image](https://github.com/user-attachments/assets/35d30cc9-a7bb-4416-856d-eb0c4f8b39d3)
When I used 0.19 version and  import Paths2CIR. It could not import name 'Paths2CIR' from 'sionna.rt'
And I found that from 0.19, 0.18, 0.17, 0.15 to 0.14 version, only 0.14 can import  'Paths2CIR' .
So I used sionna 0.14 eventually without any import error.
![image](https://github.com/user-attachments/assets/450cad4a-d5e3-432e-bb9e-19a1054ad25a)

For slashes problem. If I copy the path from the file , it have error : file does not exist!
![image](https://github.com/user-attachments/assets/cd22b380-00e7-4805-aad0-a3194ab4fbc3)
But after I change the opposite direction of slashes. It work !

@merlinND After I check the names, it success !!!  Thanks for your help!
But I encounter a new promlem
I point 2 address to show the tx and rx ,but it is the wrong point in the jupyter. I can't figure it out~
![image](https://github.com/user-attachments/assets/22df18f5-a999-4ada-afbf-feef4e771953)
![image](https://github.com/user-attachments/assets/52a1656f-1c40-4fc0-885d-dbf9590680ce)
![image](https://github.com/user-attachments/assets/328d825e-1086-462e-8123-886ad67454dc)
![image](https://github.com/user-attachments/assets/31b2c34c-7f79-4576-a821-3f8a2136502c)

In blender I point the building which is cylinder and cube building, but in the jupyter, the place is different.

![image](https://github.com/user-attachments/assets/b856b0dc-e688-4741-a9d7-480bd2b63570)

1.The scene was successfully maken, but the position is not totally correcrt.
2.I don't know why there are some problem for this code
![image](https://github.com/user-attachments/assets/f7e7ef84-1f30-458b-a163-c0208ba6ce3c)

I really appreciate all your help!!!

**Comment by merlinND** [comment]
Hello @as057923553,

About the correspondence between Blender's 3D cursor and placing TX/RX in Sionna, it might be that the coordinate system gets converted when exporting the scene. You could try swapping the Y and Z coordinates.

**Comment by Fedomer** [comment]
Hello @as057923553 ,

- at first in version >0.14 the function is moved now you calculate CIR in paths see example RT introduction in v0.19 :
`a_, tau_ = paths.cir(num_paths=75)`
I suggest you to move to the last release this is an hot project that increase a lot the functionality.

- Problem in X,Y consider to export mitsuba with Y Forward and Z Up  and it will be OK.
- If you need to map Latitude e Longitude in your scene keep your FLOOR exactly as big as needed for containing all the building and not more then with the bounding box of OSM create a function like this using geodesic lib:
`X,Y = ut.LatLon2XY(Latitude, Longitude)`
then
```
scene.add(Receiver(name='UE01',
                            position=[X,Y,1.6],
                            orientation=[0,0,0]))
```
-

**Comment by as057923553** [comment]
Thank you so much for all your help; I’ve now resolved the issue with exporting from Mitsuba. It turned out that my original exported file had the -Y and Z coordinates instead of the Y and Z coordinates.

@Fedomer , I’ll follow your advice to use version 0.19 of Sionna. I also successfully tried using version 0.19 over the weekend to explore things with Sionna. Additionally, thank you for all the valuable suggestions.

@merlinND  , thank you for teaching me so much, which has allowed me to start exploring Sionna more thoroughly. All the issues I was facing are now resolved.

Thank you both for your great support!
